### PR TITLE
10721 Customizing component deployment resources

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
- Adds a new assembly to the Managing resources title.
- Adds 4 new modules - 1 concept/overview, and 3 procedure (customizing, disabling, re-enabling).
_Note: Re-enabling steps may be temporary - due to known issue with manually removing or altering the `opendatahub.io/managed` annotation._

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: assemblies/customizing-component-deployment-resources.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: custom-resource
+
+[id="customizing-component-deployment-resources"]
```
Comment (chtyler) [inline assemblies/customizing-component-deployment-resources.adoc:7]: ID missing the _{context} at the end

[id="customizing-component-deployment-resources_{context}"]

File: modules/customizing-component-deployment-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.Procedure
+. Log in to the {openshift-platform} console as a cluster administrator.
+. In the *Administrator* perspective, click *Workloads* > *Deployments*.
+ifdef::upstream[]
+. From the *Project* drop-down list, select `opendatahub`.
+. In the *Name* column, click the name of the deployment for the component that you want to customize resources for. 
++
+[NOTE]
+====
+For more information on the deployment names for each component, see link:{odhdocshome}/managing-resources/#overview-of-component-resource-customization_custom-resource[Overview of component resource customization].
```
Comment (chtyler) [inline modules/customizing-component-deployment-resources.adoc:22]: For more information **about**

File: modules/disabling-component-resource-customization.adoc [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+[id="disabling-component-resource-customization_{context}"]
+= Disabling component resource customization
+
+[role='_abstract']
+You can disable customization of component deployment resources, and restore default  values, by adding the `opendatahub.io/managed: true` annotation to the YAML file for the component deployment.
+
+[IMPORTANT]
+====
+Manually removing or setting the `opendatahub.io/managed: true` annotation to `false` after manually adding it to the YAML file for a component deployment may cause unexpected cluster issues.
```
Comment (chtyler) [inline modules/disabling-component-resource-customization.adoc:11]: Could substitute "may" for "might".

File: modules/disabling-component-resource-customization.adoc [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+.Procedure
+. Log in to the {openshift-platform} console as a cluster administrator.
+. In the *Administrator* perspective, click *Workloads* > *Deployments*.
+ifdef::upstream[]
+. From the *Project* drop-down list, select `opendatahub`.
+. In the *Name* column, click the name of the deployment for the component to which you want to add the annotation.
++
+[NOTE]
+====
+For more information on the deployment names for each component, see link:{odhdocshome}/managing-resources/#overview-of-component-resource-customization_custom-resource[Overview of component resource customization].
```
Comment (chtyler) [inline modules/disabling-component-resource-customization.adoc:34]: "For more information about"

File: modules/disabling-component-resource-customization.adoc [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+====
+endif::[]
+ifndef::upstream[]
+. From the *Project* drop-down list, select `redhat-ods-applications`.
+. In the *Name* column, click the name of the deployment for the component to which you want to add the annotation.
++
+[NOTE]
+====
+For more information on the deployment names for each component, see link:{rhoaidocshome}{default-format-url}/customizing-component-deployment-resources/overview-of-component-resource-customization_custom-resource[Overview of component resource customization].
```
Comment (chtyler) [inline modules/disabling-component-resource-customization.adoc:43]: "For more information about"

File: modules/overview-of-component-resource-customization.adoc [hunk 6]
```diff
@@ -0,0 +1,43 @@
+:_module-type: CONCEPT
+
+[id='overview-of-component-resource-customization_{context}']
+= Overview of component resource customization
+
+[role='_abstract']
+You can customize deployment resources that are related to the {productname-long} Operator, for example, CPU and memory limits and requests. For resource customizations to persist without being overwritten by the Operator, the `opendatahub.io/managed: true` annotation must not be present in the YAML file for the component deployment. This annotation is absent by default.
```
Comment (chtyler) [inline modules/overview-of-component-resource-customization.adoc:7]: "CPU and memory limits and requests." Could this be reworded?

File: modules/reenabling-component-resource-customization.adoc [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+[id="reenabling-component-resource-customization_{context}"]
+= Re-enabling component resource customization
+
+[role='_abstract']
+You can re-enable customization of component deployment resources after manually disabling it.
+
+[IMPORTANT]
+====
+Manually removing or setting the `opendatahub.io/managed:` annotation to `false` after adding it to the YAML file for a component deployment may cause unexpected cluster issues.
```
Comment (chtyler) [inline modules/reenabling-component-resource-customization.adoc:11]: may could be changed to might

**Review by chtyler — CHANGES_REQUESTED** [review]
Just a few minor things

File: assemblies/customizing-component-deployment-resources.adoc (hunk 1 above)
Comment (grainnejenningsRH) [reply assemblies/customizing-component-deployment-resources.adoc]: Good spot, updated!

File: modules/customizing-component-deployment-resources.adoc (hunk 2 above)
Comment (grainnejenningsRH) [reply modules/customizing-component-deployment-resources.adoc]: Good catch, will update all of these instances.

File: modules/overview-of-component-resource-customization.adoc (hunk 6 above)
Comment (grainnejenningsRH) [reply modules/overview-of-component-resource-customization.adoc:7]: Discussed offline, leaving as-is.

**Review by chtyler — APPROVED** [review]

**Pull request merged**